TurkishFilter/sections/specific.txt

```diff
@@ -2203,6 +2203,8 @@ itemci.com##.header_rek
 itemci.com##a[href="https://www.astekmt2.net/"]
 hdplayer.org###adversing
 hdplayer.org###player > .alert_msg
+frmtr.com##img[src^="https://www.frmtr.com/images/x"]
+||frmtr.com/images/x*
```

Comment:
better regex in this case, less wide. Did not see your pull request, added `/frmtr.com\/images\/\w\d.gif/$domain=frmtr.com`

Reply:
If you do not add any cosmetic rule at 9cb49e1dcae05d0498f9dfbb222613f4fe0fe8ca, i will edit the commit of my PR.

Reply:
Ah, i am sorry that requested a review before i comment.

Reply:
8249dbc55a2da08cbefa320d5304d99e2142ea84